pages/index.tsx

```diff
@@ -24,6 +24,10 @@ const transition = { duration: 0.5 }
 const Home: NextPage = () => {
   return (
     <>
+
```

Comment:
Why those changes in spacing?
Can you use the auto format feature of the IDE to use prettier to format all files?

Reply:
i've run prettier reformatting on pages / styles / components and config files , i hope that's okay